### DISCUSSION TITLE
Made by nextron?

### DISCUSSION THREAD

**Discussion by saltyshiomix** [saltyshiomix/nextron#406] [Show and tell]
Please feel free to share your apps or services :)

**Comment by dgca** [comment]
Working on something right now, and the experience is amazing. Thanks so much for your work on this, I really appreciate what you've built here!

Can't wait to share the project once it's ready :)

**Comment by 0xajinkya** [comment]
I created an Ubuntu application called MINEAL (Multi-layered Improvement for Network and Endpoint Advanced Linux Security) named after my GF, it works on hardening ubuntu's security by explicitly changing the sh scripts, the experience was fun, thank you nextron guys!

**Comment by bm777** [comment]
Hey there, I made this fun, quick, and instant search engine in 3 hours! it searches in the internet and you keep working without moving the tab browser or Windows. I hope you will love it. @saltyshiomix 

[Hask AI](https://github.com/bm777/hask/) -> https://github.com/bm777/hask/

![record](https://github.com/saltyshiomix/nextron/assets/29865600/0b81f341-7918-4cee-b64d-f98700dbbe46)


Available for Mac windows, and linux!

(I'm still building another local search engine, still being cooked)

**Comment by pixelass** [comment]
We're excited to introduce Captain, an open-source application developed using Nextron. We at @blib-la, designed this "Hub" to empower your local computer or private cloud GPU with extensive AI capabilities. Captain is currently in its ALPHA stage, and we eagerly welcome your feedback: [Captain on GitHub](https://github.com/blib-la/captain).

![image](https://github.com/saltyshiomix/nextron/assets/1148334/fcde4ed3-23a0-43fe-9346-c3fc0848219c)

**Key Features of Captain:**
- **Live Painting:** Transform your sketches into stunning images in real-time.
- **Advanced Image Generation:** Utilizes the powerful Stable Diffusion XL (SDXL) for high-quality visuals.
- **Story Generation:** Currently powered by OpenAI; opt-in via API key. Integration of local large language models (LLMs) is in our roadmap.
- **Local Vector Store:** All generated content and installed apps are stored locally, enabling human language searches within the explorer or prompt windows. This feature enhances the discoverability and management of your content and applications.
- **Marketplace Integration:** Seamlessly discover and install new apps directly within Captain (coming soon).
- **Developer SDK:** Build your own applications using our robust SDK, featuring safe IPC communications and helpful components.
- **Extensive AI Features:** Explore a variety of AI functionalities.

**Technical Details:**
- Supports ONNX models via transformers.js for diverse AI model integration.
- Provides commonly used models through an embedded Python environment.
- Easy installation: Simply download the .exe file, and Captain handles the rest, including setup and dependency management.

**Privacy and Transparency Commitments:**
- **100% Transparency:** We keep you informed about all aspects of the application.
- **100% Local Data Storage:** Your data stays on your device.
- **No Tracking:** We do not track user activity.
- **No Data Collection:** We do not collect or store your personal data.

Join us in exploring the limitless possibilities with AI on your desktop today! Your insights and suggestions are invaluable as we continue to enhance Captain.

**Reply by bm777** [reply]
Oh this is cool :) @pixelass

**Reply by saltyshiomix** [reply]
Fantastic!!!

**Comment by Pavel064** [comment]
Hello!
If I use i18next My dynamic pages work perfectly in the development environment, but in the build version the dynamic pages do not work, I see a blank screen. Once you turn off localization, everything comes back to order.
Can you help me solve this problem?

**Reply by pixelass** [reply]
You canot use the regular localization from nextjs. You have to do it manually, so that it works. See https://github.com/saltyshiomix/nextron/tree/main/examples/with-next-i18next.

Also this is more of a separate discussion, since this is a showcase of Nextron apps.

**Reply by Pavel064** [reply]
yes, I did according to the example "examples/with-next-i18next.", but this example does not indicate the case of using nextron + i18next in a dynamic page, the address of which looks like this:
"pages/[locale]/account/project/[id].jsx".
the page does not work in the build of my app

**Reply by Pavel064** [reply]
Everything works well in dev mode, but in the build, after the dynamic page opens, the application breaks.
Maybe it's the next-i 18 settings.
I noticed such a moment.
When the application is running in dev mode, the folder structure for localization is as follows: http://localhost:8888/_next/data/development/en/account/project/5c1c4bb2-f4be-4768-adc2-3ee79cbfbf33.json?locale=en&id=5c1c4bb2-f4be-4768-adc2-3ee79cbfbf33 there is a translation file here

And here in the build folder is /app/_next/data/0jYM3FNTvud57HxrplYik/en/account.json

And in the wrong file, the program tries to find exactly account/project/5c1c4bb2-f4be-4768-adc2-3ee79cbfbf33.json?locale=en&id=5c1c4bb2-f4be-4768-adc2-3ee79cbfbf33 does not find and breaks

How to make it work normally?

**Reply by pixelass** [reply]
you have to make sure to provide staticPaths. (e.g. abstract the example here: https://github.com/saltyshiomix/nextron/blob/main/examples/with-next-i18next/renderer/lib/get-static.js)

So for a dynamic page: 
1. fetch all pages
2. return `getStaticPaths` function from dybamic page

**Reply by Valyukhov** [reply]
https://github.com/saltyshiomix/nextron/discussions/461 -i have the same problem

**Comment by Valyukhov** [comment]


**Reply by pixelass** [reply]
please open a new discussion. This is off topic of this discussion. I will gladly help there.
